mars/dataframe/groupby/aggregation.py

```diff
@@ -432,9 +619,15 @@ def _perform_shuffle(
         in_df: TileableType,
         out_df: TileableType,
         func_infos: ReductionSteps,
+        pivot_chunk,
```

Comment:
Could you add a type hint here, it should be a `ChunkType`.

Reply:
Fixed @qinxuye. Please confirm.